src/components/WorkshopButton/WorkshopButton.test.txt

```diff
@@ -0,0 +1,60 @@
+// Rupesh Baghmar
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { WorkshopButton, WorkshopButtonProps } from "./WorkshopButton";
+
+describe("WorkshopButton", () => {
+  const mockOnClick = jest.fn();
+
+  const renderComponent = (additionalProps?: Partial<WorkshopButtonProps>) => {
+    const defaultProps: WorkshopButtonProps = {
+      text: "Click me",
+      onClick: mockOnClick,
+    };
+
+    render(<WorkshopButton {...defaultProps} {...additionalProps} />);
+  };
+
+  it("renders button with text and default variant", () => {
```

Comment:
You're testing multiple things in one test, always separate. Because if the test fails, you don't know immediately what the issue is, is it the text or the color?